.travis.yml

```diff
@@ -0,0 +1,2 @@
+script:
+        - 
```

Comment:
It would be great to have shorter version `script: true`

Reply:
Hmm, that would work, but I don't know if that would be considered a test or not.

Reply:
It's better than no test at all, right?

Reply:
That's fair! I'll update the changes.